apps/tailwind-components/pages/Styles.other.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <h1 class="text-heading-3xl">Theme Styles</h1>
+
+  <h2 class="text-heading-2xl py-4">Colors</h2>
+
+  <h3 class="text-heading-1xl py-3">blue</h3>
+  <div class="flex flex-wrap -mb-4 mt-6">
+    <ColorTile color="blue-50" type="background" />
```

Comment:
I think these should be wrapped in an ordered list element as the order of the colours matter. The markup would like something like this:

```html
<ol class="flex flex-wrap -mb-4 mt-6">
    <li><ColorTile color="blue-50" type="background"></li>
    ...
</ol>
```